data/genesis/cappuccino.toml

```diff
@@ -1,3 +1,6 @@
+base_version = "0.1"
```

Comment:
I might add a versions table and include the base and upgrade field in it in future.